commit 1dcdf58a2b9eda2168be1c4de843bed73879a01e
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-06-04T12:43:25-07:00 (4 months ago)
    
    TRITON-1717 Firewall rules with PRIORITY attribute failed parsing during `docker inspect`

diff --git a/package.json b/package.json
index 15500e9..f6ffdda 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sdc-docker",
-  "version": "0.5.6",
+  "version": "0.5.7",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -9,7 +9,7 @@
     "docker-registry-client": "3.2.12",
     "bunyan": "1.8.8",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
-    "fwrule": "git+https://github.com/joyent/sdc-fwrule.git#d1174be",
+    "fwrule": "2.1.0",
     "imgmanifest": "3.0.0",
     "keyapi": "git+https://github.com/joyent/keyapi.git#e14b3d5",
     "libuuid": "0.2.1",
